src/test/java/org/zalando/nakadi/controller/EventTypeControllerTest.java

```diff
@@ -317,19 +317,6 @@ public void whenPostDuplicatedEventTypeReturn409() throws Exception {
     }
 
     @Test
-    public void whenPostAndTopicExistsReturn409() throws Exception {
-        final Problem expectedProblem = Problem.valueOf(Response.Status.CONFLICT, "dummy message");
-        final EventType et = buildDefaultEventType();
-        doReturn(et).when(eventTypeRepository).saveEventType(any(EventType.class));
-
-        Mockito.doThrow(new DuplicatedEventTypeNameException("dummy message")).when(topicRepository)
-                .createTopic(anyInt(), any());
-
-        postEventType(et).andExpect(status().isConflict()).andExpect(content().contentType("application/problem+json"))
-                .andExpect(content().string(matchesProblem(expectedProblem)));
-    }
```

Comment:
There is no need in it anymore, since TopicRepository does not throw DuplicatedEventTypeNameException.